.expeditor/Version_inspector/prod_default_version.json

```diff
@@ -91,7 +91,7 @@
 "rust-uninstall": "0.0.1" ,
 "rust": "1.37.0" ,
 "sequel-gem": "4.47.0" ,
-"server-open-jre": "11.0.16.1+1" ,
+"server-open-jre": "11.0.18+10" ,
```

Comment:
What is the impact of this change?

Reply:
I'm not sure.  I asked in our team channel a few days ago what this file does, but maybe nobody knew the answer?